libethash-cl/CLMiner.cpp

```diff
@@ -858,6 +854,26 @@ bool CLMiner::initEpoch_internal()
                 m_dag.push_back(
                     cl::Buffer(m_context[0], CL_MEM_READ_ONLY, (m_epochContext.dagSize) / 2));
             }
+            cllog << "Creating light cache buffer, size: "
+                  << dev::getFormattedMemory((double)m_epochContext.lightSize);
+            m_light.clear();
+            bool light_on_host = false;
+            try
+            {
+                m_light.push_back(
```

Comment:
Maybe is useful to use emplace_back instead of  push_back ?
It'd void the need to call m_light.clear() in case of error thrown (as nothing would be inserted in the vector).

Reply:
Ok